pkg/config/default-config.yml

```diff
@@ -40,6 +40,16 @@ defaultEnvironment:
   AUTH_HOST: auth
   AUTH_PORT: 9004
 
+  VOTE_HOST: vote
+  VOTE_PORT: 9013
+  VOTE_REDIS_HOST: redis
+  VOTE_REDIS_PORT: 6379
+  VOTE_DATABASE_USER: openslides
```

Comment:
Please order the database related variables like in line 30 to 35 (host, port, name, user, password, password_file).

@ostcar Can you please enable reading the db password from a file given as secret? See datastore settings here.

Reply:
@peb-adr Please add a new secret for the database password file. The secret may be called `vote_postgres_password`. You also have to update setup/setup.go at the beginning (const section) and around line 123. The randomization is an extra issue.